ProjectLighthouse/Pages/SlotsPage.cshtml.cs

```diff
@@ -22,20 +22,26 @@ public class SlotsPage : BaseLayout
     public int SlotCount;
 
     public List<Slot> Slots;
+
+    public string SearchValue;
+
     public SlotsPage([NotNull] Database database) : base(database)
     {}
 
-    public async Task<IActionResult> OnGet([FromRoute] int pageNumber)
+    public async Task<IActionResult> OnGet([FromRoute] int pageNumber, [FromQuery] string name)
```

Comment:
I'd mark this as nullable.

Reply:
Do you mean adding `[CanBeNull]`?

Reply:
That would also work, but I strongly prefer using the syntax C# comes with. You can mark it by adding `?` to the end of a type, so here it would be `[FromQuery] string? name`.

Reply:
I tried that but the IDE was correcting it to the other form so I had to go and disable it

Reply:
Yeah, Rider (I assume that's what you're using) likes to do that. How did you disable that?

Reply:
Settings -> Editor -> General -> Typing assistance then scroll down to C# and turn off annotate nullability

Reply:
Gotcha. Thanks for the tip!